packages/params/src/activePreset.ts

```diff
@@ -60,4 +65,57 @@ export const {
   INACTIVITY_PENALTY_QUOTIENT_ALTAIR,
   MIN_SLASHING_PENALTY_QUOTIENT_ALTAIR,
   PROPORTIONAL_SLASHING_MULTIPLIER_ALTAIR,
-} = preset;
+} = presets[ACTIVE_PRESET];
+
+/**
+ * Override the active preset
+ *
+ * WARNING: Lodestar libraries rely on preset values being _constant_, so the active preset must be set _before_ loading any other lodestar libraries.
+ *
+ * Only call this function if you _really_ know what you are doing.
+ */
+export function setActivePreset(presetName: PresetName): void {
+  ACTIVE_PRESET = parsePresetName(presetName);
+  const preset = presets[ACTIVE_PRESET];
+
+  // Unfortunately, there is way to dynamically update all of these variables
```

Comment:
i'm assuming you mean this?

```suggestion
  // Unfortunately, there is no way to dynamically update all of these variables
```